Makefile

```diff
@@ -40,6 +46,10 @@ update_development:
 	python -m data.update --scan=skip
 
 # downloads latest snapshot of data locally
+# Pending cloud.gov production bucket:
+# cg-4adefb86-dadb-4ecf-be3e-f1c7b4f6d084
+# Pending cloud.gov backup bucket:
+# cg-72ce4caf-d81b-4771-9b96-3624b5554587
```

Comment:
Credentials for these buckets can be found in the environment variables associated with the `gsa-ogp-pulse`/`pulse`/`pulse` app and `gsa-ogp-pulse`/`backups`/`s3-blank-app` app, respectively.

I'm currently taking the contents of our legacy AWS S3 bucket (bucket name `pulse.cio.gov`) and copying their full contents into each of the two buckets above, so that they are preserved.